docu/release_notes/2016-12-06-Technev153.html.md

```diff
@@ -0,0 +1,15 @@
+---
+date: December 20, 2016
+service: 'hybris techne'
+official_version: v1.5.3
+headline: 'Avatars and Images Pattern'
+---
+
+### New Pattern
+* New pattern that that displays avatars and images on different context. Markup and iteraction on how to Upload/Change images.
+  ![Tree screenshot](img/v1.5.3/avatars-images.png)
+
+### Minor updates
+* Simplified **Tree Pattern* HTML Markup. Removed complexity with more explicit CSS classes.
+* On the **Toogle Buttons** we now show a stylised "1/0" instead of "on/off". More universal now;  no need for translations.
```

Comment:
+* The toggle buttons now show a stylized **1/0**  instead of **on/off**. This change provides a more universal, conceptual representation for the toggle buttons, so there is no need to translate the button labels.